lab/runnerctl.py

```diff
@@ -32,13 +31,23 @@ def pytest_namespace():
     return {'halt': Halt()}
 
 
+def pytest_configure(config):
+    signal.signal(signal.SIGTERM, exit_gracefully)
```

Comment:
I guess there's no reason to use a flag to enable eh?
I can't think of a reason where someone wouldn't want this.